dasharo-security/tpm-support.robot

```diff
@@ -30,22 +30,24 @@ TPM001.001 TPM Support (firmware)
     ...    correctly and the PCRs can be accessed from the firmware.
```

Comment:
I guess because we are basing the test on cbmem, which down the line relies on firmware, it qualifies as a firmware based detection, but I agree, that documentation does not reflect the mechanism well. Do you think documentation change is enough?

Reply:
I think the `(Firmware)` and `(Ubuntu)` labels are most useful to quickly see if a test is firmware UI dependent or not, e.g., when determining a test scope. As in - it goes through EDK2 menus, won't work on SeaBIOS/heads and laptops which don't offer a serial connection. But I guess changing the documentation to clearly state it goes straight into Ubuntu should do the trick